Tests/Translation/Extractor/SmartyExtractorTest.php

```diff
@@ -1,28 +1,13 @@
 <?php
 /**
- * 2007-2018 PrestaShop
+ * This file is authored by PrestaShop SA and Contributors <contact@prestashop.com>
```

Comment:
Can we change the header of files without asking permission to all contributors of this one ?

Reply:
Yes because @sowbiba is not changing the header: he is fixing a mistake. This tool has always been licensed as MIT.